### ISSUE TITLE
Publisher's "publishing history" graph shows other publishers' editions

### ISSUE THREAD

**Issue by bencomp** [internetarchive/openlibrary#158]
Whereas the rationale for the graph labeled Publishing History is "This graph charts editions from this publisher over time", it appears that it also charts editions from other publishers who published other editions of the works that are in the first publisher's catalogue.

In pseudocode:
Let P1 be the publisher under study, who published edition E1 of work W1. Other editions of W1 are E2, E3 and E4, published by P2, P3 and P4 respectively. The graph then charts not only E1 (as per the description), but also E2, E3 and E4.

**Comment by LeadSongDog** [comment]
Bump. Example: The chart at https://openlibrary.org/publishers/Thomas_Hailes_Lacy shows issues dating back to the first-edition publication of MacBeth, which clearly predates Lacy's birth in 1809. If it can't be fixed, there should at least be a disclaimer to avoid confusing users.

**Comment by guyjeangilles** [comment]
Still a bug. @tfmorris I'm assuming we have an editions Solr to implement the algorithm in the original issue? I'm assuming either you or jdlrobson would head this.

**Comment by tfmorris** [comment]
Yes, a proper fix is dependent on indexing editions (and would probably be fixed automatically with no additional work). If someone wanted to improve things in the mean time, they could implement the [Mar 2017 suggestion](https://github.com/internetarchive/openlibrary/issues/158#issuecomment-290142480).

**Comment by guyjeangilles** [comment]
The patch until editions are indexed should be to display a disclaimer to the user so they're not confused. Assigning jdlrobson (not tagging as request) per slack discussions.

**Comment by jdlrobson** [comment]
Not a frontend issue. The frontend just just renders the data passed by the backend and that's what needs to be fixed here.

We should just remove the graph if that's not easy (also a backend change)

**Comment by mekarpeles** [comment]
This comment https://github.com/internetarchive/openlibrary/issues/158#issuecomment-552683213 suggests this is rather hard and blocked.

**Comment by tfmorris** [comment]
@mekarpeles Now that editions are being indexed in Solr, this should no longer be blocked
